ui/app/routes/evaluations/AdvancedParametersAccordion.tsx

```diff
@@ -37,29 +70,94 @@ export function AdvancedParametersAccordion({
         </AccordionTrigger>
         <AccordionContent>
           <div className="space-y-6 px-3 pt-3">
-            <Label>Inference Cache</Label>
-            <RadioGroup
-              value={inferenceCache}
-              onValueChange={setInferenceCache}
-              className="mt-2 flex gap-4"
-            >
-              <div className="flex items-center space-x-2">
-                <RadioGroupItem value="on" id={`on`} />
-                <Label htmlFor={`on`}>On</Label>
-              </div>
-              <div className="flex items-center space-x-2">
-                <RadioGroupItem value="off" id={`off`} />
-                <Label htmlFor={`off`}>Off</Label>
-              </div>
-              <div className="flex items-center space-x-2">
-                <RadioGroupItem value="read_only" id={`read_only`} />
-                <Label htmlFor={`read_only`}>Read Only</Label>
-              </div>
-              <div className="flex items-center space-x-2">
-                <RadioGroupItem value="write_only" id={`write_only`} />
-                <Label htmlFor={`write_only`}>Write Only</Label>
+            <div>
+              <Label>Inference Cache</Label>
+              <RadioGroup
+                value={inferenceCache}
+                onValueChange={setInferenceCache}
+                className="mt-2 flex gap-4"
+              >
+                <div className="flex items-center space-x-2">
+                  <RadioGroupItem value="on" id="on" />
+                  <Label htmlFor="on">On</Label>
+                </div>
+                <div className="flex items-center space-x-2">
+                  <RadioGroupItem value="off" id="off" />
+                  <Label htmlFor="off">Off</Label>
+                </div>
+                <div className="flex items-center space-x-2">
+                  <RadioGroupItem value="read_only" id="read_only" />
+                  <Label htmlFor="read_only">Read Only</Label>
+                </div>
+                <div className="flex items-center space-x-2">
+                  <RadioGroupItem value="write_only" id="write_only" />
+                  <Label htmlFor="write_only">Write Only</Label>
+                </div>
+              </RadioGroup>
+            </div>
+            {evaluatorNames.length > 0 && (
```

Comment:
can we pull this out into a separate component? should make the code a lot more readable.